src/com/mambu/apisdk/services/LinesOfCreditService.java

```diff
@@ -158,7 +180,7 @@ public LineOfCredit getLineOfCreditDetails(String lineofcreditId) throws MambuAp
 		switch (customerType) {
 		case CLIENT:
 		case GROUP:
-			return serviceExecutor.getOwnedEntities(customerType, customerId, serviceEntity, offset, limit);
+			return serviceExecutor.getOwnedEntities(customerType, customerId, serviceEntity, offset, limit, requresFullDetails);
```

Comment:
this is a very good idea, since the SDK is public the clients relays on the contract of these methods the change must come in 2 steps:
1. adding new methods supporting pagination everywhere limit and offset are used
2. deprecate the old methods using limit and offset.
And finally carry on the deprecated methods for 2-3 versions